packages/core/src/component.ts

```diff
@@ -3,34 +3,49 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import {Constructor, Provider, BoundValue} from '@loopback/context';
-import {Application} from '.';
-
-// tslint:disable:no-any
+import { Constructor, Provider, BoundValue, Context } from '@loopback/context';
+import {Server} from '.';
 
 export interface ProviderMap {
   [key: string]: Constructor<Provider<BoundValue>>;
 }
 
 export interface Component {
-  controllers?: Constructor<any>[];
+  controllers?: Array<Constructor<{}>>;
```

Comment:
I believe the current convention in TypeScript is to use `Foo[]` instead of `Array<Foo>`. Would you mind reverting this change?